cmd/tether/ops_linux.go

```diff
@@ -177,33 +178,85 @@ func (t *operations) SetupFirewall(config *tether.ExecutorConfig) error {
 			ifaceName := iface.Attrs().Name
 			log.Debugf("slot %d -> %s", endpoint.ID, ifaceName)
 
-			established.Interface = ifaceName
-			_ = established.Commit(context.TODO())
-
-			// handle the ports
-			for _, p := range endpoint.Ports {
-				// parse the port maps
-				r, err := portToRule(p)
-				if err != nil {
-					log.Errorf("can't apply port rule (%s): %s", p, err.Error())
-					continue
+			switch endpoint.Network.TrustLevel {
+			case common.Open:
+				// Accept all incoming and outgoing traffic
+				generalPolicy(netfilter.Accept)
+
+			case common.Closed:
+				// Reject all incoming and outgoing traffic
+				generalPolicy(netfilter.Drop)
```

Comment:
Currently the default policy is to drop traffic, and explicit rules need to be created to accept traffic. Should it be that the default policy is to accept, and we should explicitly reject/drop certain traffic instead?

Also, when running a container on an external network, `ifconfig` shows the external interface `eth0` and the loopback, but no bridge interface. Is there a case that a container can be attached to both a bridge and an external network? `docker network connect` does not seem to affect the interfaces either.